web/packages/teleterm/src/ui/DocumentCluster/UnifiedResources.tsx

```diff
@@ -340,6 +387,26 @@ const Resources = memo(
         params={props.queryParams}
         setParams={props.onParamsChange}
         unifiedResourcePreferencesAttempt={props.userPreferencesAttempt}
+        bulkActions={
+          props.integratedAccessRequests.supported === 'yes'
+            ? [
+                {
+                  key: 'requestAccess',
+                  Icon: icons.AddCircle,
```

Comment:
Another thing I noticed during testing, as seen in the video that I posted above, is that when the window gets narrow enough, those extra actions in the bar above resources get reduced to just icons, but they lack any kind of `title` so it's hard to tell what they're supposed to do.

<img width="553" alt="resources-bar" src="https://github.com/user-attachments/assets/23ed754b-cb8c-4e89-95b9-c9c9d54da805">

Reply:
Hmm, adding in the `tooltip` works but now there is a tooltip in the full text mode as well. I could have swore this was a case that was handled before. I'll fix this separately.